### ISSUE TITLE
localstack on windows does not open port 4566

### ISSUE THREAD

**Issue by chadfaurie** [localstack/localstack#2989]
# Type of request: This is a ...

[X] bug report
[ ] feature request

# Detailed description
Runnning localstack start on windows does not open port 4566

## Expected behavior

`localstack start` should open ports 4566-4620

## Actual behavior

only opens ports 4567-4620

# Steps to reproduce

## Command used to start LocalStack

`localstack start`

this produces this output in powershell

```
docker run -it -e DEFAULT_REGION="us-east-1" -e LOCALSTACK_HOSTNAME="localhost" -e TEST_AWS_ACCOUNT_ID="000000000000" --rm --privileged --name localstack_main -p 4567-4620:4567-4620 -p 8080-8081:8080-8081  -v "C:\Users\chadf\AppData\Local\Temp\localstack:/tmp/localstack" -v "/var/run/docker.sock:/var/run/docker.sock" -e DOCKER_HOST="unix:///var/run/docker.sock" -e HOST_TMP_FOLDER="C:\Users\chadf\AppData\Local\Temp\localstack" "localstack/localstack"
```

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

N/A

**Comment by vinifig** [comment]
I'm facing this even in linux in the latest version

I fixed it using the docker image `0.11.4`

**Comment by whummer** [comment]
Thanks for reporting @chadfaurie @vinifig . Can you please give it another try with the latest version of the CLI:
```
pip install --upgrade --no-cache localstack localstack-client
```
Please let us know if the problem persists with the latest release version `0.11.5`. Thanks
